SeleniumTests/LoginTests.cs

```diff
@@ -140,5 +146,30 @@ public void An_Administrator_Logs_In_Successfully()
         //** NOTE:  After completeing this section of the Selenium Assignment, if you understand the code above and what you've done
         //          you now know more than 70% of the functions you will use in Selenium automation, there are other methods of course, 
         //          but these are the most commonly used.
+    
+
+        // login helper functopn
+        private void do_login(string user_name, string password)
+        {
+            _driver.FindElement(By.Id("userName")).SendKeys(user_name);
+            _driver.FindElement(By.Id("password")).SendKeys(password);
+            var submitButtonXPath = "/html/body/div/div[3]/div[1]/div[1]/div/div[2]/form/input[3]"; //**** PUT THE XPATH VALUE HERE!!!
+            _driver.FindElement(By.XPath(submitButtonXPath)).Click();
+        }
+
+        // div scraper function
+        private int get_number()
+        {
+            String scraped = _driver.FindElement(By.XPath("/html/body/div/div[3]/div[1]/div[1]/div/div[2]/div")).Text;
```

Comment:
XPath is the last option you want to use (sometimes it's all you have), but in general it's the most fragile.  WHEN, the ui changes, either because of a design change or because the page is data driven and the data changes, the xpath will break.

In this case, on line 163, the div that displays the number contains a class='msg'.  Use the FindElement(By.ClassName("msg"))  (it might be findelements?  but then get element[0]).